tests/slurm-tests/run_all.sh

```
@@ -11,6 +11,8 @@ python tests/slurm-tests/super_49b_evals/run_test.py --cluster $CLUSTER --worksp
 # sleep 10
 python tests/slurm-tests/qwen3_4b_evals/run_test.py --cluster $CLUSTER --workspace /workspace/nemo-skills-slurm-ci/$RUN_NAME/qwen3_4b_evals --expname_prefix qwen3_4b_evals_$RUN_NAME
 # sleep 10
+python tests/slurm-tests/super_120b_aime25/run_test.py --cluster $CLUSTER --workspace /workspace/nemo-skills-slurm-ci/$RUN_NAME/super_120b_aime25 --expname_prefix super_120b_aime25_$RUN_NAME
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)